test/unit/application.spec.js

```diff
@@ -36,22 +37,6 @@ describe('Marionette Application', function() {
     });
   });
 
-  describe('#start', function() {
```

Comment:
I think this could move back.  `start` triggers `onBeforeStart` so it's first in the stack

Reply:
Seems fair, just tried to match the order of the codebase file.

Reply:
well yes but:
```js
1.  start(options) {
2.   this.triggerMethod('before:start', this, options);  //onBeforeStart
3.   this.triggerMethod('start', this, options);  //onStart
```